src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -503,93 +516,109 @@ const BaseCourseCard = ({
+      >
+        <Icon src={Info} size="md" />
+        <span>{alertText}</span>
+      </Stack>
     );
   };
 
   const dropdownMenuItems = getDropdownMenuItems();
-  const isExecutiveEducation2UCourse = EXECUTIVE_EDUCATION_COURSE_MODES.includes(mode);
 
   return (
     <div className={classNames(
-      'dashboard-course-card py-3 border-bottom mb-2',
-      { 'exec-ed-course-card rounded-lg p-3 text-light-100': isExecutiveEducation2UCourse },
-      { 'mb-3': (isCanceledAssignment || isExpiredAssignment) },
+      'dashboard-course-card-wrapper border-bottom',
+      { 'py-4': !isExecutiveEducation2UCourse },
     )}
     >
-      {isLoading ? (
-        <>
-          <div className="sr-only">Loading...</div>
-          <Skeleton height={50} />
-        </>
-      )
-        : (
+      <div className={classNames(
+        'dashboard-course-card',
+        { 'exec-ed-course-card text-light-200 p-3 my-3 rounded shadow-sm': isExecutiveEducation2UCourse },
+      )}
+      >
+        {isLoading ? (
           <>
-            <div className="d-flex">
-              <div className="flex-grow-1 mr-4 mb-3">
-                {renderMicroMastersTitle()}
-                <Stack gap={2} direction="horizontal" className="align-items-start justify-content-between mb-1">
-                  <h4 className="course-title mb-0">
-                    <CourseTitleComponent
-                      className={classNames('h3', { 'text-white': isExecutiveEducation2UCourse })}
-                      destination={externalCourseLink ? linkToCourse : null}
-                      to={!externalCourseLink ? linkToCourse : null}
-                    >
-                      {title}
-                    </CourseTitleComponent>
-                  </h4>
-                  {renderBadge()}
-                </Stack>
-                {renderOrganizationName()}
-              </div>
-              {renderSettingsDropdown(dropdownMenuItems)}
-            </div>
-            {renderCourseInfoOutline()}
-            {renderButtons()}
-            {renderChildren()}
-            <Row className="course-misc-text">
-              <Col className={`${isExecutiveEducation2UCourse ? 'text-light-300' : 'text-gray'}`}>
-                {renderMiscText()}
-                {renderAdditionalInfoOutline()}
-                {hasViewCertificateLink && renderViewCertificateText()}
-              </Col>
-            </Row>
-            {(isCanceledAssignment || isExpiredAssignment) && (
-              <Row
-                className={classNames({
-                  'mt-4 assignment-alert-row': isExecutiveEducation2UCourse,
-                  'mt-2 pl-2': !isExecutiveEducation2UCourse,
-                })}
-              >
-                {renderAssignmentAlert()}
-              </Row>
-            )}
-            {renderEmailSettingsModal()}
-            {renderUnenrollModal()}
+            <div className="sr-only">Loading...</div>
+            <Skeleton height={100} />
```

Comment:
nit/curious: I do wonder if we should ensure the `Skeleton` height here is the same as the `Skeleton` height introduced in the parent `Suspense` fallback ([source](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/350413391b1482a99d9081fc7c17acadb020180f/src/components/dashboard/main-content/course-enrollments/CourseSection.jsx#L114)).